mappings/net/minecraft/server/world/ChunkPrioritySorter.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3900 net/minecraft/server/world/ChunkTaskPrioritySystem
+CLASS net/minecraft/class_3900 net/minecraft/server/world/ChunkPrioritySorter
```

Comment:
This is specifically sorting chunk task priorities- why the rename?

Reply:
It sorts, so it's a sorter (rather than a "priority system".
What it sorts is Chunks, so it's a chunk sorter.
What it sorts by is chunk priority so it's a ChunkPrioritySorter.

I found this name to be a little clearer than what we had before.

Reply:
It is responsible for much more than just simply sorting chunks, though- and it is very much tied to the tasks that it executes.